README.md

```diff
@@ -186,6 +186,10 @@ See LICENCE.
 
 ## Change Log
 
+####3.0.3
```

Comment:
I'd prefer the minor version number to be incremented, i.e. 3.1.0, as new functionality is being introduced.
"Exposed TO_EXP_NEG and TO_EXP_POS as Big.E_NEG and Big.E_POS."